lib/travis/build/addons/deploy/script.rb

```diff
@@ -142,12 +142,12 @@ def run
 
             def install
               sh.if "$(rvm use $(travis_internal_ruby) do ruby -e \"puts RUBY_VERSION\") = 1.9*" do
-                cmd(dpl_install_command(WANT_18), echo: false, assert: !allow_failure, timing: true)
+                cmd(dpl_install_command(WANT_18), echo: true, assert: !allow_failure, timing: true)
```

Comment:
Is echoing enabled on purpose here or it's a remainder of your tests?

Reply:
It's done on purpose; so that it is a little clearer what is happening.